src/cosmosdb-preview/azext_cosmosdb_preview/commands.py

```diff
@@ -100,3 +111,19 @@ def load_command_table(self, _):
         g.command('list', 'list_sql_role_assignments')
         g.show_command('show', 'get_sql_role_assignment')
         g.command('delete', 'delete_sql_role_assignment', confirmation=True)
+
+    with self.command_group('cassandra-mi cluster', cosmosdb_managed_cassandra_cluster_sdk, client_factory=cf_cassandra_cluster, is_preview=True) as g:
+        g.custom_command('create', 'cli_cosmosdb_managed_cassandra_cluster_create')
+        g.custom_command('update', 'cli_cosmosdb_managed_cassandra_cluster_update')
+        g.command('node status', 'fetch_node_status')
+        g.command('list-by-subscription', 'list_by_subscription')
```

Comment:
list will be a single command, in there you can provide different parameter sets for customer. i.e if subscription is provided then you should call the list by subscription sdk method, if rg then the corresponding rg method. If both are provided then throw error.